runtime/runtime-params-estimator/src/main.rs

```diff
@@ -71,6 +71,8 @@ struct CliArgs {
     /// Print extra debug information
     #[clap(long, multiple(true), possible_values=&["io", "rocksdb", "least-squares"])]
     debug: Vec<String>,
+    #[clap(long)]
+    tracing_span_tree: bool,
```

Comment:
:facepalm: it literally didn't occur to me that we can make this a command-line argument.

Reply:
Could you add a doc comment here though? So that it gets to the `--help` message